### ISSUE TITLE
compute test set accuracy

### ISSUE THREAD

**Issue by jsherrah** [RockStarCoders/alienMarkovNetworks#28]
Now we have used training to train the classifier, and validation to select hyper-parameters.  We want to get the best result on the test set, so now re-train the classifier using those optimal parameters on the training-plus-validation set.
